RadioGalaxyZoo/index.html

```diff
@@ -0,0 +1,79 @@
+<!doctype html>
+<!--[if lt IE 7]>      <html class="no-js lt-ie9 lt-ie8 lt-ie7" lang=""> <![endif]-->
+<!--[if IE 7]>         <html class="no-js lt-ie9 lt-ie8" lang=""> <![endif]-->
+<!--[if IE 8]>         <html class="no-js lt-ie9" lang=""> <![endif]-->
+<!--[if gt IE 8]><!--> <html class="no-js" lang=""> <!--<![endif]-->
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+        <title>Radio Galaxy Zoo: Archive</title>
+        <meta name="description" content="">
```

Comment:
Do we want a description here for search engines?

Reply:
Yes, I can add something in here. Probably something general like "Zooniverse Radio Galaxy Zoo: Archived Page"